Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp

```diff
@@ -12881,6 +12884,19 @@ IGNORE_CLANG_WARNINGS_END
         setJSValue(vmCall(pointerType(), operationMapSet, weakPointer(globalObject), map, key, value, hash));
     }
 
+    void compileMapOrSetDelete()
+    {
+        JSGlobalObject* globalObject = m_graph.globalObjectFor(m_origin.semantic);
+        LValue mapOrSet = lowSetObject(m_node->child1());
```

Comment:
lowMapObject should be selected for Map

Reply:
Fixed. Not sure how this was a speedup with such a bug. It's definitely faster with this fix though.